contrib/network/connection_balance/dlb/source/connection_balancer_impl.cc

```diff
@@ -37,35 +35,17 @@ DlbConnectionBalanceFactory::createConnectionBalancerFromProto(
         "please decrease the number of threads by `--concurrency`");
   }
 
+  const int& config_id = dlb_config.id();
```

Comment:
Sorry for not being clear earlier.
The [id](https://github.com/envoyproxy/envoy/blob/1ad89a900b9239539820d9ff5c322414efa4fd07/api/contrib/envoy/extensions/network/connection_balance/dlb/v3alpha/dlb.proto#L32) is supposed to be unsigned int. Why is it being converted to a signed int?

Reply:
Sure, I will update it.